provisioning/service/samples/Getting Started/IndividualEnrollmentX509Sample/IndividualEnrollmentX509Sample.cs

```diff
@@ -0,0 +1,105 @@
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Samples
+{
+    internal class IndividualEnrollmentX509Sample
+    {
+        private string _registrationId;
+        private readonly ProvisioningServiceClient _provisioningServiceClient;
+        private readonly X509Certificate2 _issuerCertificate;
+
+        // Optional parameters
+        private string _deviceId;
+
+        private const ProvisioningStatus OptionalProvisioningStatus = ProvisioningStatus.Enabled;
+        private readonly DeviceCapabilities _optionalEdgeCapabilityEnabled = new() { IotEdge = true };
+        private readonly DeviceCapabilities _optionalEdgeCapabilityDisabled = new() { IotEdge = false };
+
+        public IndividualEnrollmentX509Sample(ProvisioningServiceClient provisioningServiceClient, X509Certificate2 issuerCertificate, string deviceId, string registrationId)
+        {
+            _provisioningServiceClient = provisioningServiceClient;
+            _issuerCertificate = issuerCertificate;
+            _deviceId = deviceId;
+            _registrationId = registrationId;
+        }
+
+        public async Task RunSampleAsync()
+        {
+            await QueryIndividualEnrollmentsAsync();
+            await CreateIndividualEnrollmentX509Async();
+            await UpdateIndividualEnrollmentAsync();
+            await DeleteIndividualEnrollmentAsync();
+        }
+
+        public async Task QueryIndividualEnrollmentsAsync()
+        {
+            Console.WriteLine("\nCreating a query for enrollments...");
+            Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery("SELECT * FROM enrollments");
+            while (query.HasNext())
+            {
+                QueryResult queryResult = await query.NextAsync();
+                Console.WriteLine(queryResult);
+            }
+        }
+
+        public async Task CreateIndividualEnrollmentX509Async()
+        {
+            Console.WriteLine("\nCreating a new individualEnrollment...");
+            X509Attestation _x509 = X509Attestation.CreateFromClientCertificates(_issuerCertificate);
+            var individualEnrollment = new IndividualEnrollment(
+                _registrationId, _x509)
+            {
+                //The following parameters are optional:
+                DeviceId = _deviceId,
+                ProvisioningStatus = OptionalProvisioningStatus,
+                Capabilities = _optionalEdgeCapabilityEnabled,
+                InitialTwinState = new TwinState(
+                    tags: null,
+                    desiredProperties: new TwinCollection()
+                    {
+                        ["Brand"] = "Contoso",
+                        ["Model"] = "SSC4",
+                        ["Color"] = "White",
+                    })
+            };
+            Console.WriteLine(individualEnrollment);
```

Comment:
Same as above. I don't expect this will print anything meaningful. What do you see?